Sources/Purchasing/StoreKit1/ProductsFetcherSK1.swift

```diff
@@ -86,12 +86,16 @@ final class ProductsFetcherSK1: NSObject {
 
     func products(withIdentifiers identifiers: Set<String>,
                   completion: @escaping (Result<Set<SK1StoreProduct>, PurchasesError>) -> Void) {
-        self.sk1Products(withIdentifiers: identifiers) { skProducts in
-            let result = skProducts
-                .map { Set($0.map(SK1StoreProduct.init)) }
-
-            completion(result)
-        }
+        TimingUtil.measureAndLogIfTooSlow(
```

Comment:
I'd be great to actually test that this class actually logs if this is too slow, to future-proof so that a refactor can't accidentally get rid of the logging